source/server/lsp-servers.adb

```diff
@@ -50,6 +50,10 @@ package body LSP.Servers is
    New_Line : constant String :=
      (Ada.Characters.Latin_1.CR, Ada.Characters.Latin_1.LF);
 
+   Filesystem_Monitoring_Trace : constant GNATCOLL.Traces.Trace_Handle :=
```

Comment:
Can we use a configuration item instead of the trace? It will be easier for users of VS Code to change it. Also it would be mo visible to VS Code users.

Reply:
I'll do this later: now we need to urgently deactivate this so that we can get GS wavefronts

Reply:
Agreed